helm/csi-isilon/values.yaml

```diff
@@ -332,3 +332,8 @@ podmon:
   #    - "--mode=node"
   #    - "--leaderelection=false"
   #    - "--driver-config-params=/csi-isilon-config-params/driver-config-params.yaml"
+
+# Set the below parameter for setting the path size limit
+# Deafult value None
+# Examples: 128, 256 (Depends on the driver type and allowed value for path size limit)
+maxPathLen:
```

Comment:
In all other cases we have set a default value.
In the scenario this is left empty, what is the default in the code. You can use the same value here as default

Reply:
The format of comment which specifies what this does, also needs to be changed. Please look at description of other variable in their comments.

Reply:
is this volume size or complete path size of the given volume ? or path or full address for the end LUN ?

Reply:
It is the complete path size of the given volume

Reply:
> In all other cases we have set a default value. In the scenario this is left empty, what is the default in the code. You can use the same value here as default

Set the default value

Reply:
> The format of comment which specifies what this does, also needs to be changed. Please look at description of other variable in their comments.

This has been fixed